src/api/comment/content-types/comment/schema.json

```diff
@@ -24,7 +24,7 @@
     "author": {
       "type": "string"
     },
-    "authorId": {
```

Comment:
Quick search in the frontend repo shows that it's not using this field, so the renaming is safe.